CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Updated internal data handling that have improved performance and reduced memory usage.
+- Updated data structure of the problem instance file:
+	-- To extract all the values in the first line of the instance, additional fields i.e. (`best_sol`, `sol_time_bfgs`, `num_frac_values`) are added.
+	--  The field`solution_vector` is added to extract vector of the solution to the problem instance at the end of the new instance file compared the old data structure
```

Comment:
To answer your questions, let's keep in mind that we are extending the extraction of input data from a new problem instance. These fields are added because end users will have an option for further investigation.
Anyway, thank you for the suggestion: I modified your answer and rewrite both terms

Reply:
Thanks for the catch. I updated that line based on your suggestion. Please check out the commit